pytorch_pfn_extras/training/manager.py

```diff
@@ -519,12 +520,34 @@ def state_dict(
                                  for name in self._optimizers}
         to_save['extensions'] = {name: self._extensions[name].state_dict()
                                  for name in self._extensions}
+        import pytorch_pfn_extras as ppe
+        to_save['ppe_version'] = ppe.__version__
```

Comment:
Use `pytorch_pfn_extras` to avoid recursive import issue.

```suggestion
        to_save['ppe_version'] = pytorch_pfn_extras.__version__
```